src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -38,6 +38,8 @@ public sealed class LegacyPackageReferenceProject : BuildIntegratedNuGetProject
         private string _projectName;
         private string _projectUniqueName;
         private string _projectFullPath;
+        private Dictionary<NuGetFramework, List<Utility.InstalledPackageCache>> _installedPackages = new Dictionary<NuGetFramework, List<Utility.InstalledPackageCache>>();
```

Comment:
Same here, qualify InstalledPackageCache.

Reply:
In LegacyPackageReference, there's always only 1 framework, so you can consider avoiding the 2 level cache.